nav2_amcl_localizer/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.5)
-project(nav2_amcl)
+project(nav2_amcl_localizer)
```

Comment:
Suffix `_localizer` is redundant.

Reply:
Also a very long name -- nav2_amcl seems sufficient for a name